### PR TITLE
add help section to redirects

### PR DESCRIPTION
**Pull request by asmega**

### CODE REVIEW

**Comment by jonalport** [comment]
Nice collapsables :+1:

**Comment by asmega** [comment]
I couldn't config the icon classes in collapsible annoyingly

**Comment by jonalport** [comment]
Mmm I remember having to override them in the SASS

File: app/views/redirects/help.html.erb [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+          <% end %>
+        </div>
+      </div>
+    </div>
+</div>
+
+<div class="l-panel-content l-redirects-help">
+  <div class="l-constrained">
+    <section data-dough-component="Collapsable">
+    <h2 data-dough-collapsable-trigger="1"><span class="fa fa-plus"></span> Redirect an article, video and other pages served through Contento</h2>
```
Comment (benbarnett) [inline app/views/redirects/help.html.erb]: Indentation slightly out of whack

Comment (asmega) [inline app/views/redirects/help.html.erb]: fixed

Comment (asmega) [inline app/views/redirects/help.html.erb]: fixed again. there was another indentation issue i fixed

**Comment by benbarnett** [comment]
:+1:

**Pull request merged**